player/lua/osc.lua

```diff
@@ -2978,6 +2982,7 @@ end)
 
 validate_user_opts()
 visibility_mode(user_opts.visibility, true)
+set_osc_styles()
```

Comment:
put it also in the `read_options` callback for runtime updates.

Reply:
Inside `opt.readoptions()`? (pushed)

Reply:
Yes, and like I said, put it immediately after `validate_user_opts`. Important to do it before `request_tick`.